hc/front/views.py

```diff
@@ -60,6 +63,68 @@ def my_checks(request):
     return render(request, "front/my_checks.html", ctx)
 
 
+@login_required
+def failed_checks(request):
```

Comment:
Well i would have preferred the LoginRequiredMixin; i guess the decorator is okay for simplicity's sake!